src/data/multiset/basic.lean

```diff
@@ -1911,6 +1913,15 @@ by rw [inter_comm, repeat_inter, min_comm]
 
 end
 
+@[ext]
```

Comment:
Are you sure this will ever fire? I'm afraid `ext` will only ever call `add_hom.ext`.

Reply:
Yes, it's used below.